src/utils/list.ts

```diff
@@ -18,4 +21,6 @@ export const mapDictionary = <T, S>(obj:Dictionary<T>, keyMapper:Mapper<string,
         .map(([k,v]) => ({newKey: keyMapper(k), newValue: valueMapper(v)}))
         .reduce((soFar, {newKey, newValue}) => ({...soFar, [newKey]: newValue}), {});
 
-export type Mapper<T, S> = (t:T) => S
+export const zip = <T, S>(ts:Array<T>, ss:Array<S>):Maybe<Array<Pair<T, S>>> => {
```

Comment:
Just curious why we return a maybe type here.

Reply:
i guess because lodash itself can return undefined from the underlying zip operation... but it would be nice to enforce that this operation always returns a result.